vmdb/app/views/layouts/gtl/_list.html.haml

```diff
@@ -0,0 +1,353 @@
+                      - click = "cfmeDynatree_activateNode('settings_tree', 'msc-#{to_cid(row['id'])}');"
+                    - elsif x_active_tree == :schedules_tree
+                      - click = "cfmeDynatree_activateNode('schedules_tree', 'msc-#{to_cid(row['id'])}');"
+                  - when "MiqReportResult"
+                    - if x_active_tree == :savedreports_tree
+                      - if x_node .split('-').length == 1
+                        - click = remote_function(:loading => "miqSparkle(true);",
+                          :complete => "miqSparkle(false);",
+                          :url      => {:action => 'get_report', :id => "rr-#{to_cid(row['id'])}"})
+                      - else
+                        - click = "cfmeDynatree_activateNode('savedreports_tree', 'rr-#{to_cid(row['id'])}');"
+                    - elsif x_active_tree == :reports_tree
+                      - click = "cfmeDynatree_activateNode('reports_tree', '#{x_node}_rr-#{to_cid(row['id'])}');"
+                  - when "MiqWidget"
+                    - typ = WIDGET_CONTENT_TYPE.invert[row['content_type']]
+                    - click = "cfmeDynatree_activateNode('widgets_tree', 'xx-#{typ}_-#{to_cid(row['id'])}');"
+                  - when "IsoDatastore"
+                    - click = "cfmeDynatree_activateNode('iso_datastores_tree', 'isd-#{to_cid(row['id'])}');"
+                  - when "PxeServer"
+                    - click = "cfmeDynatree_activateNode('pxe_servers_tree', 'ps-#{to_cid(row['id'])}');"
+                  - when "PxeImageType"
+                    - click = "cfmeDynatree_activateNode('pxe_image_types_tree', 'pit-#{to_cid(row['id'])}');"
+                  - when "CustomizationTemplate"
+                    - click = "cfmeDynatree_activateNode('customization_templates_tree', 'ct-#{to_cid(row['id'])}');"
+                  - when "ScanItemSet"
+                    - click = "cfmeDynatree_activateNode('settings_tree', 'sis-#{to_cid(row['id'])}');"
+                  - when "ChargebackRate"
+                    - click = "cfmeDynatree_activateNode('#{x_active_tree}', '#{x_node}_cr-#{to_cid(row['id'])}');"
+                  - when "MiqDialog"
+                    - click = "cfmeDynatree_activateNode('old_dialogs_tree', '#{X_TREE_NODE_PREFIXES.invert["MiqDialog"]}-#{to_cid(row['id'])}');"
+                  - when "MiqWorker"
+                    - click = remote_function(:url => {:action => 'diagnostics_worker_selected', :id => "#{row['id']}"})
+                  - else
+                    - if view.db == "Action"
+                      - click = "DoNav('#{url_for_db("miq_action")}');"
+                    - else
+                      - if @edit && @edit[:hostitems]
+                        - click = "DoNav('/host/edit/?selected_host=#{@id}');"
+                      - else
+                        - click = "DoNav('#{url_for_db(view.db)}');"
+            - if col == "message"
```

Comment:
the 2 `col ==` cam nbe 1 condition the same way as below the 2nd endif. But a `case` statement end the `col` before it would be better